src/actions/app.js

```diff
@@ -151,3 +151,19 @@ export function reportTrackThreadHeight(
 export function dismissNewlyPublished(): Action {
   return { type: 'DISMISS_NEWLY_PUBLISHED' };
 }
+
+export function startDragging(): Action {
```

Comment:
I generally recommend adding code comments to the start of all the action creators, as these are the public API that are consumed in the app.